wg-structured-logging/annual-report-2021.md

```diff
@@ -5,50 +5,50 @@
 1. What work did the WG do this year that should be highlighted?
    For example, artifacts, reports, white papers produced this year.
 
-   -
-   -
-   -
+   - Graduated [Structured Logging](https://github.com/kubernetes/enhancements/issues/1602) to Beta
+   - Graduated [Derecation of klog specific flags](https://github.com/kubernetes/enhancements/issues/2845) to Beta
+   - Created draft proposal for [Contextual logging](https://docs.google.com/document/d/1FaDj3_MvLWSuUtlVapgfTUglhYjBem1LNcpdd_OyBwU/edit?usp=sharing)
```

Comment:
The KEP PR was also created in 2021, I think that's a better link than the Google doc. The prototype also was ready ->

```suggestion
   - Created prototype and KEP for [Contextual logging](https://github.com/kubernetes/enhancements/pull/3078)
```

Reply:
agree